client/my-sites/site-settings/form-writing.jsx

```diff
@@ -141,12 +142,13 @@ class SiteSettingsFormWriting extends Component {
 				{ jetpackSettingsUI &&
 					jetpackVersionSupportsLazyImages && (
 						<div>
-							{ this.renderSectionHeader( translate( 'Speed up your site' ), false ) }
+							{ this.renderSectionHeader( translate( 'Performance & speed' ), false ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Performance' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).